include/taskolib/Tag.h

```diff
@@ -106,6 +106,25 @@ class Tag
         return a.name_ >= b.name_;
     }
 
+    /// Concatenate a string and a tag.
+    friend std::string operator+(const std::string& lhs, const Tag& rhs)
+    {
+        return lhs + rhs.name_;
+    }
+
+    /// Concatenate a tag and a string.
+    friend std::string operator+(const Tag& lhs, const std::string& rhs)
+    {
+        return lhs.name_ + rhs;
+    }
+
+    /// Concatenate a string and a tag.
+    friend std::string& operator+=(std::string& lhs, const Tag& rhs)
```

Comment:
Hmm ... what about the symmetric operator `operator+={const Tag& lhs, std::string rhs)`? But the test case looks a little bit wired:

```c++
TEST_CASE("operator+=(Tag, string)", "[Tag]")
{
    Tag tag{"Hello "};
    tag += "world";
    REQUIRE(tag.string() == "hello world");
}
```

Reply:
I do not really think that makes sense. Right now, a `Tag` is practically immutable: You can assign another `Tag` to it, but it cannot be modified in any other way. This new `operator+=(Tag, string)` would also have to throw if the string contains illegal characters.